File: PALINDROME/Palindrome Numbers.js
function palindromeCheck(num) {
    palindrome = num.toString()
    var newstr = '';
    var length = palindrome.length - 1;
    var count = length;
    while (count >= 0) {
        newstr += palindrome[count].toLowerCase();
        count -= 1;
    }
    return newstr == palindrome.toLowerCase();
}

console.log(palindromeCheck("Radar"))

File: PALINDROME/Palindrome Numbers.py
def palindromeCheck(num):
  palindrome = len(num)
  newstr = ''
  length = len(palindrome) - 1
  count = length
  while count >= 0:
    newstr += palindrome[count].lower()
    count -= 1 
  return newstr == palindrome.lower()

print(palindromeCheck("Radar"))